internal/utility/fileposition/column.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/google/osv-scanner/internal/cachedregexp"
 )
 
-var wordRe = cachedregexp.MustCompile(`[^\s(\n|\r|\r\n)]+`)
+var wordRe = cachedregexp.MustCompile(`[^\s\r\n]+`)
```

Comment:
why do we remove this possibility ? How much does it work with CR, LF, CRLF files ?